includes/admin/give-metabox-functions.php

```diff
@@ -370,6 +374,43 @@ class="give-select-chosen give-chosen-settings"
 	<?php
 }
 
+/**
+ * Output a label field
+ * Note: only for internal use.
+ *
+ * @param array $field         {
+ *                              Optional. Array of text input field arguments.
+ *
+ * @type string $id            Field ID. Default ''.
+ * @type string $wrapper_class CSS class to use for wrapper of input field. Default ''.
+ * @type string $value         Value of input field. Default ''.
+ * @type string $title         Text to display. HTML supported Default ''.
+ * @type string $before_field  Text/HTML to add before input field. Default ''.
+ * @type string $after_field   Text/HTML to add after input field. Default ''.
+ * }
```

Comment:
Do we have a Field class we can use, instead?

Reply:
No, we don't have any Field class in Give core. I have added an additional field support in Give core using the way the support for fields is already implemented.

Reply:
I see. We'll want to refactor this at some point, then. If an object has a predictable shape, then it should be a class rather than a mutational array — which has no constraints on its shape. Don't worry about it, now, but keep this in mind down the road, please. 😃